.storybook/manager.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { addons } from '@storybook/addons';
+import { create } from '@storybook/theming';
+import { zdFontFamilySystem, zdFontFamilyMonospace } from '@zendeskgarden/css-variables';
```

Comment:
🤔 we should probably be getting these from `react-components` theming, not CSS variables.